tests/newSequences.mapp

```diff
@@ -0,0 +1,61 @@
+# name                    number of elements       address          size           bar    width   fracbits    signed
```

Comment:
It seems the named sequences are only appearing in the minimalistic test case which merely counts the number of registers after parsing. I would suggest to change the channel names in this map file to something arbitrary. That change should not make any difference right now.

Also, I would change the order of channel entries in at least one register, such that the lines/entries are not ordered with ascending addresses. This again should not change the test result (since the channels shall be numbered with ascending addresses, not following the line ordering in the map file).

Reply:
Changed in new map file